crates/nu-command/src/stor/insert.rs

```diff
@@ -40,20 +40,29 @@ impl Command for StorInsert {
         vec![Example {
             description: "Insert data the in-memory sqlite database using a data-record of column-name and column-value pairs",
             example: "stor insert --table-name nudb --data-record {bool1: true, int1: 5, float1: 1.1, str1: fdncred, datetime1: 2023-04-17}",
-            result: None,
-        }]
+            result: None
+            },
+            Example {
+            description: "Insert data into the in-memory sqlite database by pipelining a record of column-name and column-value pairs",
+            example: "{bool1: true, int1: 5, float1: 1.1, str1: fdncred, datetime1: 2023-04-17} | stor insert --table-name nudb",
+            result: None
+            }]
     }
 
     fn run(
         &self,
         engine_state: &EngineState,
         stack: &mut Stack,
         call: &Call,
-        _input: PipelineData,
+        input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
         let span = call.head;
         let table_name: Option<String> = call.get_flag(engine_state, stack, "table-name")?;
-        let columns: Option<Record> = call.get_flag(engine_state, stack, "data-record")?;
+        let columns: Option<Record> = match input {
```

Comment:
We should probably check if both `input` and `--data-record` parameter is used and provide an error message. Seems crazy, I know, but people will do it.

Reply:
Would `ShellError::IncompatibleParameters` be the right thing to return from this?

Reply:
ya, that sounds good.